### ISSUE TITLE
Action not raising non zero exit code on latest version

### ISSUE THREAD

**Issue by cswilliams** [reviewdog/action-rubocop#126]
We noticed in the latest version with this configuration:
```
      - name: rubocop
        uses: reviewdog/action-rubocop@5e23bb67d79c93e5eb45bdae6d08b04052afec35 # v2.19.1
        with:
          rubocop_version: gemfile
          rubocop_extensions: rubocop-rails:gemfile rubocop-rspec:gemfile rubocop-rspec_rails:gemfile rubocop-performance:gemfile rubocop-capybara:gemfile rubocop-factory_bot:gemfile
          reporter: github-pr-check
          fail_on_error: true
          rubocop_flags: --config .rubocop.yml
          filter_mode: nofilter

```
that warnings are no longer causing the action to raise a non zero exit code. We also see this warning:
```
time=2024-10-03T20:49:01.820Z level=WARN msg="reviewdog: -fail-on-error is deprecated. Use -fail-level=any, or -fail-level=error for github-[pr-]check reporter instead. See also https://github.com/reviewdog/reviewdog/blob/master/CHANGELOG.md"
```

When we rolled back to `v2.18.1` of the action, warnings are causing a non-zero exit code again.

I'm wondering if https://github.com/reviewdog/action-rubocop/pull/124 needs to be merged in and then we need to set fail-level any to resolve this going forward?

**Comment by massongit** [comment]
https://github.com/reviewdog/action-rubocop/pull/124 is merged.
